pkgs/development/tools/analysis/codeql/default.nix

```
@@ -1,26 +1,18 @@
-{ lib, stdenv
-, fetchzip
-, zlib
-, xorg
-, freetype
-, alsa-lib
-, jdk11
-, curl
-, lttng-ust
-, autoPatchelfHook
-}:
+{ lib, stdenv, fetchzip, zlib, xorg, freetype, alsa-lib, jdk11, curl, lttng-ust
```

Remove unused inputs:

```suggestion
{ lib, stdenv, fetchzip, zlib, xorg, freetype, jdk11, curl
```